### PR TITLE
Add payment overview article

### PR DESCRIPTION
**Pull request by SariSaar**
- Add an article consolidating Flex payment information. 
- Reorder payments related background articles in sidebar to group them together
- Reorder payments related cookbook articles to a more logical flow
- Update article about commissions to reflect the current recommended way to define commissions.

### CODE REVIEW

File: src/docs/background/payments-overview/index.md [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+capabilities. In this article, you will learn about the Flex default
+payment integration, implemented with Stripe, as well as the
+alternatives in case you decide to not use Stripe on your marketplace.
+
+## Marketplace payment flow
+
+In a nutshell, a basic payment flow in a marketplace contains five
+significant steps:
+
+![Payment gateway logic](./payment-flow.png)
```
Comment (SariSaar) [inline src/docs/background/payments-overview/index.md:22]: This section is now almost duplicate with the 3rd party payment gateway article beginning. Any feedback welcome re: should one of the duplicates be removed or trimmed somehow!

File: src/docs/background/payments-overview/index.md [hunk 2]
```diff
@@ -0,0 +1,397 @@
...
+
+### Step 5: Provider payout
+
+If everything in the transaction went right and the customer received
+the agreed service, the money from the payment gateway will be
+eventually paid out to the provider.
+
+## Stripe default integration
+
+In the default Flex implementation, the process described above is
```
Comment (lyyder) [inline src/docs/background/payments-overview/index.md:75]: Should we phrase this somehow like _In the default transaction process in Flex, the process described..._ to emphasize that by modifying the transaction process, the payment integration and flow can be changed? Or does _implementation_ also refer to FTW?

File: src/docs/background/payments-overview/index.md [hunk 3]
```diff
@@ -0,0 +1,397 @@
...
+implemented using Stripe. The integration uses
+[Stripe Custom Connect accounts](https://stripe.com/docs/connect/custom-accounts)
+as the provider’s payout account. The customer can checkout using a
+payment card, and they can also save their payment method for future
+use. The integration uses
+[Stripe destination charges](https://stripe.com/docs/connect/destination-charges)
+to collect the payment to the platform’s account first. Once the
+transaction is successfully over, the provider’s share of the payment is
+paid through the provider’s Custom Connect account to the bank account
+they provided upon onboarding.
```
Comment (lyyder) [inline src/docs/background/payments-overview/index.md:85]: IIRC the amount is first transferred to the provider's Connect account (using destination charges?). From there possible commission is paid to the operator's Stripe platform account using Stripe's application fee. Pay out then pays the remaining part to the provider's bank account from the Connect account.

File: src/docs/background/payments-overview/index.md [hunk 4]
```diff
@@ -0,0 +1,397 @@
...
+[Stripe's own documentation](https://stripe.com/docs/connect/required-verification-information).
+
+#### 2. Customer checkout
+
+When the customer initiates a transaction in the Flex default
+transaction processes, Flex creates a
+[PaymentIntent](/background/payment-intents/) for the total price of the
+transaction. Once the PaymentIntent is confirmed, Stripe preauthorizes
+the sum from the customer's payment method. In other words, the sum does
+not leave the customer's account, but it is still reserved and not
```
Comment (lyyder) [inline src/docs/background/payments-overview/index.md:115]: s/account/credit card

Or can money leave credit card? Would it be more along the lines of "the sum is not paid from the customer's credit card, but it still..."

File: src/docs/background/payments-overview/index.md [hunk 5]
```diff
@@ -0,0 +1,397 @@
...
+
+- [:action/stripe-refund-payment](/references/transaction-process-actions/#actionstripe-refund-payment)
+
+#### 5. Provider payout
+
+Once the booking has completed successfully, the provider's payout is
+paid to the bank account that is linked to their Custom Connect account.
+Depending on how the
+[transaction's line items](/background/pricing/#line-items) have been
+defined, the platform can take a commission of the price from either the
```
Comment (lyyder) [inline src/docs/background/payments-overview/index.md:166]: Commissions could also be mentioned in step 3. as during the payment intent capture, the commissions are paid to the platform account from the Connect account. The way that it's implemented is that commissions are paid using a Stripe concept called _application fee_. When Flex invokes Stripe to capture a payment intent, the application fee is passed in the request as a param and therefore Stripe knows to transfer the amount to the Connect account and from that deduct the application fee and move that to the platform account.

File: src/docs/background/payments-overview/index.md [hunk 6]
```diff
@@ -0,0 +1,397 @@
...
+provider, the customer, or both. The platform is also responsible for
+paying all [Stripe fees](https://stripe.com/en-fi/connect/pricing)
+related to the Custom Connect account usage, so the commissions must be
+defined to cover those expenses as well.
+
+It is important to note that the platform can
+[hold funds on their Stripe account](https://stripe.com/docs/connect/account-balances#holding-funds)
```
Comment (lyyder) [inline src/docs/background/payments-overview/index.md:173]: "the platform" -> "on their account"

Should we say something like "Stripe can hold captured funds for up to 90 days..."

File: src/docs/background/payments-overview/index.md [hunk 7]
```diff
@@ -0,0 +1,397 @@
...
+for up to 90 days - for exceptions see the linked Stripe documentation.
+In other words, the payout must be triggered no more than 90 days after
+the PaymentIntent is created. This means that for booking times
+exceeding 90 days, the process needs to be modified.
+
+<extrainfo title="Manual or automatic payout?">
+In Stripe terms, the Flex integration uses manual payouts. This means
+that Stripe does not automatically pay out the funds on the platform
+account e.g. daily or weekly, and instead the platform controls the payout
```
Comment (lyyder) [inline src/docs/background/payments-overview/index.md:182]: s/platform/Flex

The operator's Stripe account is known as "platform account". Therefore, I think here platform can accidentally be confused with Stripe, or what do you think. We could also include here that the transaction process in Flex controls the payout schedule, but that may be too accurate for this paragraph so maybe stating that Flex controls the payout schedule is enough.

File: src/docs/background/payments-overview/index.md [hunk 8]
```diff
@@ -0,0 +1,397 @@
...
+transaction process related to each transaction in
+[Flex Console](https://flex-console.sharetribe.com/) > Manage >
+Transactions.
+
+The transaction process also controls the automatic
+[email notifications](references/email-templates/) sent at different
+stages of the transaction flow. When you make changes to the transaction
+process, be sure to also update the wording and logic of the
+notifications for a consistent user experience for your marketplace
+customers and providers.
```
Comment (lyyder) [inline src/docs/background/payments-overview/index.md:219]: 👏

**Review by lyyder — COMMENTED** [review]
I think this article looks really good and is a long due improvement to Flex payments documentation.

File: src/docs/background/payments-overview/index.md (hunk 2 above)
Comment (SariSaar) [reply src/docs/background/payments-overview/index.md]: Good point, clarified to `In the default Flex transaction process and FTW templates`

File: src/docs/background/payments-overview/index.md (hunk 5 above)
Comment (SariSaar) [reply src/docs/background/payments-overview/index.md:181]: Mention added!

File: src/docs/background/payments-overview/index.md (hunk 3 above)
Comment (SariSaar) [reply src/docs/background/payments-overview/index.md]: Clarified!

File: src/docs/background/payments-overview/index.md (hunk 4 above)
Comment (SariSaar) [reply src/docs/background/payments-overview/index.md]: Wording fixed

File: src/docs/background/payments-overview/index.md (hunk 6 above)
Comment (SariSaar) [reply src/docs/background/payments-overview/index.md]: Wording fixed

File: src/docs/background/payments-overview/index.md (hunk 7 above)
Comment (SariSaar) [reply src/docs/background/payments-overview/index.md]: Wording clarified

File: src/docs/background/payments-overview/index.md [hunk 9]
```diff
@@ -0,0 +1,397 @@
...
+[FTW Hourly (Yogatime)](https://github.com/sharetribe/ftw-hourly), as
+well as any custom client application you may be using.
+
+#### Automatic off-session payments
+
+Another notable approach to modifying the payment timeline in Flex is
+the
+[off-session payment pattern](/background/off-session-payments-in-transaction-process/).
+In an off-session payment, the customer checkout and provider acceptance
+happen when the customer books the listing, but the payment takes place
```
[the remark this answers is not in the document]
Comment (SariSaar) [reply src/docs/background/payments-overview/index.md]: Fixed wording here to remove references to booking

File: src/docs/background/payments-overview/index.md [hunk 10]
```diff
@@ -0,0 +1,397 @@
...
+  Also pay attention that in test environment, you need to use the
+  Stripe keys starting with `sk_test` and `pk_test`, and you will also
+  need to use
+  [Stripe's test payout details](https://stripe.com/docs/connect/testing#payouts)
+  and
+  [test payment methods](https://stripe.com/docs/testing#payment-intents-api)
+  with those test keys. In production with real payment methods, you
+  will need to use the keys starting with `sk_live` and `pk_live`. Also
+  check that the keys you are using match the keys in the Stripe
+  Console. You can "roll" i.e. refresh the keys if necessary and enter
```
[the remark this answers is not in the document]
Comment (SariSaar) [reply src/docs/background/payments-overview/index.md]: True, changed :)

File: src/docs/background/payments-overview/index.md [hunk 11]
```diff
@@ -0,0 +1,397 @@
...
+  single button click.
+- The customer can save their payment method to Flex either when
+  purchasing a listing, or on a separate Payment Methods page.
+
+## Frequently asked questions
+
+### Where can I use Stripe?
+
+In order to use Stripe on your marketplace, your platform needs to be in
+a Stripe-supported country. The platform country then determines in
```
[the remark this answers is not in the document]
Comment (SariSaar) [reply src/docs/background/payments-overview/index.md]: Good question! The operator defines the account country in Stripe Dashboard > Settings > Account settings > Country. Added a parenthetical comment to the text about this as well.

**Comment by SariSaar** [comment]
TODO: Update dates on articles after review done!

File: src/docs/background/payments-overview/index.md [hunk 12]
```diff
@@ -0,0 +1,446 @@
+---
+title: Payments in Flex
+slug: payments-overview
+updated: 2021-11-18
+category: background
+ingress:
+  This article introduces how payments work in Flex in general, and
+  describes the default Stripe payment gateway integration
+published: false
```
Comment (shareoc) [inline src/docs/background/payments-overview/index.md:9]: 🤔 should this be `published: true` or did I miss something?

Comment (SariSaar) [reply src/docs/background/payments-overview/index.md]: You're right, I have no idea why it was left as `false`. Toggled!

**Pull request merged**